projects/plugins/protect/src/js/routes/firewall/firewall-admin-section-hero.tsx

```diff
@@ -84,16 +84,18 @@ const FirewallAdminSectionHero = () => {
 	}, [ status ] );
 
 	return (
-		<AdminSectionHero
-			main={
-				<>
-					<Status status={ 'on' === status ? 'active' : 'inactive' } label={ statusLabel } />
-					<AdminSectionHero.Heading>{ heading }</AdminSectionHero.Heading>
-					<AdminSectionHero.Subheading>{ subheading }</AdminSectionHero.Subheading>
-				</>
-			}
-			secondary={ wafSupported && <FirewallStatCards /> }
-		/>
+		<AdminSectionHero>
+			<AdminSectionHero.Main>
+				<Status status={ 'on' === status ? 'active' : 'inactive' } label={ statusLabel } mb={ 2 } />
```

Comment:
Looks like this was actually intended to have a 16px margin-bottom, but `mb` does not appear to be an assignable prop.

Reply:
Changed to using a custom style for the margin: https://github.com/Automattic/jetpack/pull/40516/commits/01586324b772b4be63a0b3e74f235074025f4c67 :+1: